problem1.py

```diff
@@ -1 +1,18 @@
+print ("********** Welcome to Celsius to Fahrenheit temperature converter **********")
 
+celsius = float(input("Insert Temperature in Celsius here "))
```

Comment:
include a colon at the end of your input string to form a more appropriate prompt
```
celsius = float(input("Insert Temperature in Celsius here:  "))
```